--- udebPackage/usr/share/debootstrap/scripts/gutsy.orig	2014-11-10 18:21:37.000000000 +0000
+++ udebPackage/usr/share/debootstrap/scripts/gutsy	2015-04-15 09:28:44.290437000 +0000
@@ -112,7 +112,8 @@

 	p; progress $baseprog $bases INSTCORE "Installing core packages" #2
 	ln -sf mawk "$TARGET/usr/bin/awk"
-	x_core_install base-files base-passwd
+	x_core_install base-passwd
+	x_core_install base-files
 	p; progress $baseprog $bases INSTCORE "Installing core packages" #3
 	x_core_install dpkg

