*** ks.cfg.orig	2016-01-25 08:56:16.406241267 +0100
--- ks.cfg	2016-01-25 14:31:11.417761494 +0100
***************
*** 98,104 ****
  default_drive=`echo ${drives} ${removable_drives} | awk '{print $1}'`
  
  installdrive=${installdrive:-undefined}
! forceformat=${forceformat:-no}
  for I in $(cat /proc/cmdline); do
    case "$I" in
      *=*)
--- 98,104 ----
  default_drive=`echo ${drives} ${removable_drives} | awk '{print $1}'`
  
  installdrive=${installdrive:-undefined}
! forceformat=${forceformat:-yes}
  for I in $(cat /proc/cmdline); do
    case "$I" in
      *=*)
