*** build/repos/nailgun/fuel_agent/fuel_agent/manager.py.orig	Thu Sep 24 11:08:38 2015
--- build/repos/nailgun/fuel_agent/fuel_agent/manager.py	Thu Sep 24 11:10:25 2015
***************
*** 541,546 ****
--- 541,552 ----
                      fs_options=fs.options,
                      fs_label=fs.label,
                      dev=str(fs.device))
+                 if fs.type == 'ext4':
+                     LOG.debug('Trying to disable journaling for ext4 '
+                               'in order to speed up the build')
+                     utils.execute('tune2fs', '-O', '^has_journal',
+                                   str(fs.device))
+ 
  
              # mounting all images into chroot tree
              self.mount_target(chroot, treat_mtab=False, pseudo=False)
***************
*** 631,636 ****
--- 637,652 ----
              self.umount_target(chroot, pseudo=False, try_lazy_umount=False)
  
              for image in self.driver.image_scheme.images:
+                 # find fs with the same loop device object
+                 # as image.target_device
+                 fs = self.driver.partition_scheme.fs_by_device(
+                     image.target_device)
+ 
+                 if fs.type == 'ext4':
+                     LOG.debug('Trying to re-enable journaling for ext4')
+                     utils.execute('tune2fs', '-O', 'has_journal',
+                                   str(fs.device))
+ 
                  LOG.debug('Deattaching loop device from file: %s',
                            image.img_tmp_file)
                  bu.deattach_loop(str(image.target_device))
