From: Fuel OPNFV <fuel@opnfv.org>
Date: Mon, 13 Jun 2016 22:23:57 +0200
Subject: OPNFV: Additions to bootstrap_admin_node.sh

---
diff --git a/iso/bootstrap_admin_node.sh b/iso/bootstrap_admin_node.sh
index 3197c91..e035145 100755
--- a/iso/bootstrap_admin_node.sh
+++ b/iso/bootstrap_admin_node.sh
@@ -339,8 +339,22 @@ fuelmenu --save-only --iface=$ADMIN_INTERFACE || fail
 set +x
 echo "Done!"

+### OPNFV addition BEGIN
+shopt -s nullglob
+for script in /opt/opnfv/bootstrap/pre.d/*.sh
+do
+  echo "Pre script: $script" >> /root/pre.log 2>&1
+  $script >> /root/pre.log 2>&1
+done
+shopt -u nullglob
+### OPNFV addition END
+
+# Enable sshd
+systemctl enable sshd
+systemctl start sshd
+
 if [[ "$showmenu" == "yes" || "$showmenu" == "YES" ]]; then
-  fuelmenu || fail
+  fuelmenu
 else
   # Give user 15 seconds to enter fuelmenu or else continue
   echo
@@ -360,9 +374,10 @@ else
   fi
 fi

+# OPNFV: Disabled to speedup installation in offline env.
 # Enable online base MOS repos (security, updates) if we run an ISO installation
-[ -f /etc/fuel_build_id ] && \
-  yum-config-manager --enable mos${FUEL_RELEASE}-security mos${FUEL_RELEASE}-updates --save
+#[ -f /etc/fuel_build_id ] && \
+#  yum-config-manager --enable mos${FUEL_RELEASE}-security mos${FUEL_RELEASE}-updates --save

 if [ ! -f "${ASTUTE_YAML}" ]; then
   echo ${fuelmenu_fail_message}
@@ -377,7 +392,5 @@ if [ ! -f /etc/fuel_build_id ]; then

-# Enable sshd
-systemctl enable sshd
-systemctl start sshd
+systemctl restart sshd

 # Enable iptables
 systemctl enable iptables.service
@@ -532,6 +545,16 @@ systemctl start ntpd

 bash /etc/rc.local

+### OPNFV addition BEGIN
+shopt -s nullglob
+for script in /opt/opnfv/bootstrap/post.d/*.sh
+do
+  echo "Post script: $script" >> /root/post.log 2>&1
+  $script >> /root/post.log 2>&1
+done
+shopt -u nullglob
+### OPNFV addition END
+
 if [ "`get_bootstrap_skip`" = "False" ]; then
   build_ubuntu_bootstrap bs_status || true
 else
@@ -571,8 +571,8 @@
 fi

 #Check if repo is accessible
-echo "Checking for access to updates repository..."
-repourl=$(yum repolist all -v | awk '{if ($1 ~ "baseurl" && $3 ~ "updates") print $3}' | head -1)
+echo "Checking for access to updates repository/mirrorlist..."
+repourl=$(yum repolist all -v | awk '{if ($1 ~ "baseurl|mirrors" && $3 ~ "updates") print $3}' | head -1)
 if urlaccesscheck check "$repourl" ; then
   UPDATE_ISSUES=0
 else
