From: Fuel OPNFV <fuel@opnfv.org>
Date: Mon, 13 Jun 2016 22:23:57 +0200
Subject: OPNFV: Additions to bootstrap_admin_node.sh

---
diff --git a/iso/bootstrap_admin_node.sh b/iso/bootstrap_admin_node.sh
index 3197c91..e035145 100755
--- a/iso/bootstrap_admin_node.sh
+++ b/iso/bootstrap_admin_node.sh
@@ -339,8 +339,22 @@ fuelmenu --save-only --iface=$ADMIN_INTERFACE || fail
 set +x
 echo "Done!"
 
+### OPNFV addition BEGIN
+shopt -s nullglob
+for script in /opt/opnfv/bootstrap/pre.d/*.sh
+do
+  echo "Pre script: $script" >> /root/pre.log 2>&1
+  $script >> /root/pre.log 2>&1
+done
+shopt -u nullglob
+### OPNFV addition END
+
+# Enable sshd
+systemctl enable sshd
+systemctl start sshd
+
 if [[ "$showmenu" == "yes" || "$showmenu" == "YES" ]]; then
-  fuelmenu || fail
+  fuelmenu
 else
   # Give user 15 seconds to enter fuelmenu or else continue
   echo
@@ -360,9 +374,10 @@ else
   fi
 fi
 
+# OPNFV: Disabled to speedup installation in offline env.
 # Enable online base MOS repos (security, updates) if we run an ISO installation
-[ -f /etc/fuel_build_id ] && \
-  yum-config-manager --enable mos${FUEL_RELEASE}-security mos${FUEL_RELEASE}-updates --save
+#[ -f /etc/fuel_build_id ] && \
+#  yum-config-manager --enable mos${FUEL_RELEASE}-security mos${FUEL_RELEASE}-updates --save
 
 if [ ! -f "${ASTUTE_YAML}" ]; then
   echo ${fuelmenu_fail_message}
@@ -377,9 +392,7 @@ if [ ! -f /etc/fuel_build_id ]; then
 [ ! -f /etc/fuel_build_id ] && \
   sed -i "s|127.0.0.1:8080/ubuntu/x86_64|mirror.fuel-infra.org/mos-repos/ubuntu/${FUEL_RELEASE}|g" "${ASTUTE_YAML}"
 
-# Enable sshd
-systemctl enable sshd
-systemctl start sshd
+systemctl reload sshd
 
 # Enable iptables
 systemctl enable iptables.service
@@ -532,6 +545,16 @@ systemctl start ntpd
 
 bash /etc/rc.local
 
+### OPNFV addition BEGIN
+shopt -s nullglob
+for script in /opt/opnfv/bootstrap/post.d/*.sh
+do
+  echo "Post script: $script" >> /root/post.log 2>&1
+  $script >> /root/post.log 2>&1
+done
+shopt -u nullglob
+### OPNFV addition END
+
 if [ "`get_bootstrap_skip`" = "False" ]; then
   build_ubuntu_bootstrap bs_status || true
 else
