*** bootstrap_admin_node.sh.orig	Mon May 30 06:31:38 2016
--- bootstrap_admin_node.sh	Mon May 30 06:35:11 2016
***************
*** 339,346 ****
  set +x
  echo "Done!"
  
  if [[ "$showmenu" == "yes" || "$showmenu" == "YES" ]]; then
!   fuelmenu || fail
  else
    # Give user 15 seconds to enter fuelmenu or else continue
    echo
--- 339,360 ----
  set +x
  echo "Done!"
  
+ ### OPNFV addition BEGIN
+ shopt -s nullglob
+ for script in /opt/opnfv/bootstrap/pre.d/*.sh
+ do
+   echo "Pre script: $script" >> /root/pre.log 2>&1
+   $script >> /root/pre.log 2>&1
+ done
+ shopt -u nullglob
+ ### OPNFV addition END
+ 
+ # Enable sshd
+ systemctl enable sshd
+ systemctl start sshd
+ 
  if [[ "$showmenu" == "yes" || "$showmenu" == "YES" ]]; then
!   fuelmenu
  else
    # Give user 15 seconds to enter fuelmenu or else continue
    echo
***************
*** 374,382 ****
  [ ! -f /etc/fuel_build_id ] && \
    sed -i "s|127.0.0.1:8080/ubuntu/x86_64|mirror.fuel-infra.org/mos-repos/ubuntu/${FUEL_RELEASE}|g" "${ASTUTE_YAML}"
  
! # Enable sshd
! systemctl enable sshd
! systemctl start sshd
  
  # Enable iptables
  systemctl enable iptables.service
--- 388,394 ----
  [ ! -f /etc/fuel_build_id ] && \
    sed -i "s|127.0.0.1:8080/ubuntu/x86_64|mirror.fuel-infra.org/mos-repos/ubuntu/${FUEL_RELEASE}|g" "${ASTUTE_YAML}"
  
! systemctl reload sshd
  
  # Enable iptables
  systemctl enable iptables.service
***************
*** 529,534 ****
--- 541,556 ----
  
  bash /etc/rc.local
  
+ ### OPNFV addition BEGIN
+ shopt -s nullglob
+ for script in /opt/opnfv/bootstrap/post.d/*.sh
+ do
+   echo "Post script: $script" >> /root/post.log 2>&1
+   $script >> /root/post.log 2>&1
+ done
+ shopt -u nullglob
+ ### OPNFV addition END
+ 
  if [ "`get_bootstrap_skip`" = "False" ]; then
    build_ubuntu_bootstrap bs_status || true
  else
