*** bootstrap_admin_node.sh.orig	2016-05-09 13:28:06.512745370 +0200
--- bootstrap_admin_node.sh	2016-05-09 14:13:48.011605839 +0200
***************
*** 339,344 ****
--- 339,358 ----
  set +x
  echo "Done!"
  
+ ### OPNFV addition BEGIN
+ shopt -s nullglob
+ for script in /opt/opnfv/bootstrap/pre.d/*.sh
+ do
+   echo "Pre script: $script" >> /root/pre.log 2>&1
+   $script >> /root/pre.log 2>&1
+ done
+ shopt -u nullglob
+ ### OPNFV addition END
+ 
+ # Enable sshd
+ systemctl enable sshd
+ systemctl start sshd
+ 
  if [[ "$showmenu" == "yes" || "$showmenu" == "YES" ]]; then
    fuelmenu || fail
  else
***************
*** 377,385 ****
    ln -s ${wwwdir}/${OPENSTACK_VERSION}/ubuntu ${wwwdir}/ubuntu
  fi
  
! # Enable sshd
! systemctl enable sshd
! systemctl start sshd
  
  # Enable iptables
  systemctl enable iptables.service
--- 391,397 ----
    ln -s ${wwwdir}/${OPENSTACK_VERSION}/ubuntu ${wwwdir}/ubuntu
  fi
  
! systemctl reload sshd
  
  # Enable iptables
  systemctl enable iptables.service
***************
*** 532,537 ****
--- 544,559 ----
  
  bash /etc/rc.local
  
+ ### OPNFV addition BEGIN
+ shopt -s nullglob
+ for script in /opt/opnfv/bootstrap/post.d/*.sh
+ do
+   echo "Post script: $script" >> /root/post.log 2>&1
+   $script >> /root/post.log 2>&1
+ done
+ shopt -u nullglob
+ ### OPNFV addition END
+ 
  if [ "`get_bootstrap_skip`" = "False" ]; then
    build_ubuntu_bootstrap bs_status || true
  else
