*** prepare-build-env.sh.orig	Mon Oct 19 13:28:30 2015
--- prepare-build-env.sh	Mon Oct 19 13:29:10 2015
***************
*** 87,92 ****
--- 87,93 ----
      echo "Docker is running."
    else
      echo "Process is not running, starting it..."
+     sudo sh -c 'echo DOCKER_OPTS=\"--dns 8.8.8.8 --dns 8.8.4.4 --bip 172.45.0.1/24\" >> /etc/default/docker*'
      sudo service docker start
    fi
  else
***************
*** 102,108 ****
    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
    # Install docker
    sudo apt-get update
!   sudo apt-get -y install lxc-docker-1.5.0
  fi
  
  # Install software
--- 103,110 ----
    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
    # Install docker
    sudo apt-get update
!   sudo sh -c 'echo DOCKER_OPTS=\"--dns 8.8.8.8 --dns 8.8.4.4 --bip 172.45.0.1/24\" > /etc/default/docker'
!   sudo apt-get -y -o Dpkg::Options::="--force-confold" install lxc-docker-1.5.0
  fi
  
  # Install software
