*** bootstrap/module.mk.orig	2016-04-22 15:30:34.872016256 +0200
--- bootstrap/module.mk	2016-04-22 15:31:32.241993497 +0200
***************
*** 1,3 ****
--- 1,6 ----
+ CPIOARGS = $(shell cpio --extract-over-symlinks -o </dev/null \
+ >/dev/null 2>&1 && echo "--extract-over-symlinks")
+ 
  .PHONY: bootstrap clean clean-bootstrap
  
  bootstrap: $(ARTS_DIR)/$(BOOTSTRAP_ART_NAME)
***************
*** 231,241 ****
  
  	# Installing kernel modules
  	find $(LOCAL_MIRROR_CENTOS_OS_BASEURL) -name '$(KERNEL_PATTERN)' | xargs rpm2cpio | \
! 		( cd $(INITRAMROOT); sudo cpio -idm './lib/modules/*' './boot/vmlinuz*' )
  	find $(LOCAL_MIRROR_CENTOS_OS_BASEURL) -name '$(KERNEL_FIRMWARE_PATTERN)' | xargs rpm2cpio | \
! 		( cd $(INITRAMROOT); sudo cpio -idm './lib/firmware/*' )
  	find $(LOCAL_MIRROR_CENTOS_OS_BASEURL) -name 'libmlx4*' | xargs rpm2cpio | \
! 		( cd $(INITRAMROOT); sudo cpio -idm './etc/*' './usr/lib64/*' )
  	for version in `ls -1 $(INITRAMROOT)/lib/modules`; do \
  		sudo depmod -b $(INITRAMROOT) $$version; \
  	done
--- 234,244 ----
  
  	# Installing kernel modules
  	find $(LOCAL_MIRROR_CENTOS_OS_BASEURL) -name '$(KERNEL_PATTERN)' | xargs rpm2cpio | \
! 		( cd $(INITRAMROOT); sudo cpio $(CPIOARGS) -idm './lib/modules/*' './boot/vmlinuz*' )
  	find $(LOCAL_MIRROR_CENTOS_OS_BASEURL) -name '$(KERNEL_FIRMWARE_PATTERN)' | xargs rpm2cpio | \
! 		( cd $(INITRAMROOT); sudo cpio $(CPIOARGS) -idm './lib/firmware/*' )
  	find $(LOCAL_MIRROR_CENTOS_OS_BASEURL) -name 'libmlx4*' | xargs rpm2cpio | \
! 		( cd $(INITRAMROOT); sudo cpio $(CPIOARGS) -idm './etc/*' './usr/lib64/*' )
  	for version in `ls -1 $(INITRAMROOT)/lib/modules`; do \
  		sudo depmod -b $(INITRAMROOT) $$version; \
  	done
