*** prepare-build-env.sh.orig	Tue Sep  8 10:29:08 2015
--- prepare-build-env.sh	Tue Sep  8 10:30:21 2015
***************
*** 43,49 ****
      GEMPKG="ruby ruby-dev"
      sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 1D2B45A2
      echo "deb http://mirror.fuel-infra.org/devops/ubuntu/ ./" | sudo tee /etc/apt/sources.list.d/fuel-devops.list
!     sudo apt-get update && sudo apt-get -y install nodejs nodejs-legacy npm
      ;;
  
    precise)
--- 43,49 ----
      GEMPKG="ruby ruby-dev"
      sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 1D2B45A2
      echo "deb http://mirror.fuel-infra.org/devops/ubuntu/ ./" | sudo tee /etc/apt/sources.list.d/fuel-devops.list
!     sudo apt-get update && sudo apt-get -y install nodejs nodejs-legacy npm dosfstools xorriso
      ;;
  
    precise)
