*** ks.cfg.orig	2016-04-26 10:10:11.481587709 +0200
--- ks.cfg	2016-04-26 10:10:11.481587709 +0200
***************
*** 579,584 ****
--- 579,592 ----
  
  EOF
  
+ ######### OPNFV addition BEGIN ############
+ # Copy data into /opt/opnfv
+ # TODO: This ought to be a package instead!
+ mkdir -p /opt/opnfv
+ cp -r ${SOURCE}/opnfv /opt
+ cp ${SOURCE}/gitinfo.txt /
+ ######### OPNFV addition END ############
+ 
  # Unmounting source
  umount -f ${SOURCE}
  rm -rf ${SOURCE}
***************
*** 647,650 ****
      /etc/sysconfig/network-scripts/ifcfg-${ifname%%:*} > \
      /mnt/sysimage/etc/sysconfig/network-scripts/ifcfg-${adminif}
  fi
! %end
\ No newline at end of file
--- 655,658 ----
      /etc/sysconfig/network-scripts/ifcfg-${ifname%%:*} > \
      /mnt/sysimage/etc/sysconfig/network-scripts/ifcfg-${adminif}
  fi
! %end
