*** config.mk.orig	Thu Jan  7 23:30:38 2016
--- config.mk	Thu Jan  7 23:32:36 2016
***************
*** 153,159 ****
  # 'msk', 'srt', 'usa', 'hrk', 'cz'
  # Setting any other value or removing of this variable will cause
  # download of all the packages directly from internet
! USE_MIRROR?=ext
  
  ifeq ($(USE_MIRROR),ext)
  MIRROR_FUEL?=http://mirror.fuel-infra.org/mos-repos/centos/$(PRODUCT_NAME)$(PRODUCT_VERSION)-centos$(CENTOS_MAJOR)-fuel/os/x86_64/
--- 153,159 ----
  # 'msk', 'srt', 'usa', 'hrk', 'cz'
  # Setting any other value or removing of this variable will cause
  # download of all the packages directly from internet
! USE_MIRROR?=cz
  
  ifeq ($(USE_MIRROR),ext)
  MIRROR_FUEL?=http://mirror.fuel-infra.org/mos-repos/centos/$(PRODUCT_NAME)$(PRODUCT_VERSION)-centos$(CENTOS_MAJOR)-fuel/os/x86_64/
