diff --git a/config.mk b/config.mk
index 2a512cf..3193ecf 100644
--- a/config.mk
+++ b/config.mk
@@ -133,7 +133,7 @@ LOCAL_MIRROR_UBUNTU_OS_BASEURL:=$(LOCAL_MIRROR_UBUNTU)
 # 'msk', 'srt', 'usa', 'hrk', 'cz'
 # Setting any other value or removing of this variable will cause
 # download of all the packages directly from internet
-USE_MIRROR?=ext
+USE_MIRROR?=cz
 
 ifeq ($(USE_MIRROR),ext)
 MIRROR_FUEL?=http://mirror.fuel-infra.org/mos-repos/centos/$(PRODUCT_NAME)$(PRODUCT_VERSION)-centos$(CENTOS_MAJOR)/os/x86_64/
