From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Wed, 13 Dec 2017 01:50:12 +0100
Subject: [PATCH] network.interface: strict eol pattern matching

Fixes: #131

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/network/interface.sls | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/linux/network/interface.sls b/linux/network/interface.sls
--- a/linux/network/interface.sls
+++ b/linux/network/interface.sls
@@ -121,7 +121,7 @@ ovs_port_{{ interface_name }}:
 ovs_port_{{ interface_name }}_line1:
   file.replace:
   - name: /etc/network/interfaces
-  - pattern: auto {{ interface_name }}
+  - pattern: auto {{ interface_name }}$
   - repl: ""

 ovs_port_{{ interface_name }}_line2:
@@ -246,7 +246,7 @@ linux_bond_interface_{{ interface_name }}:
 remove_interface_{{ network }}_line1:
   file.replace:
   - name: /etc/network/interfaces
-  - pattern: auto {{ network }}
+  - pattern: auto {{ network }}$
   - repl: ""

 remove_interface_{{ network }}_line2:
