*** ks.cfg.orig	Wed Apr 15 21:47:09 2015
--- ks.cfg	Wed Apr 15 21:47:24 2015
***************
*** 35,41 ****
  default_drive=`echo ${drives} ${removable_drives} | awk '{print $1}'`
  
  installdrive="undefined"
! forceformat="no"
  for I in `cat /proc/cmdline`; do case "$I" in *=*) eval $I;; esac ; done
  
  set ${drives} ${removable_drives}
--- 35,41 ----
  default_drive=`echo ${drives} ${removable_drives} | awk '{print $1}'`
  
  installdrive="undefined"
! forceformat="yes"
  for I in `cat /proc/cmdline`; do case "$I" in *=*) eval $I;; esac ; done
  
  set ${drives} ${removable_drives}
