From: "Stefan K. Berg" <sfb@consultron.com>
Date: Fri, 4 Nov 2016 16:06:42 +0100
Subject: [PATCH] Patches for OPNFV

---
 iso/bootstrap_admin_node.sh | 30 +++++++++++++++++++++++++++---
 1 file changed, 27 insertions(+), 3 deletions(-)

diff --git a/iso/bootstrap_admin_node.sh b/iso/bootstrap_admin_node.sh
index dbcac5f..472e418 100755
--- a/iso/bootstrap_admin_node.sh
+++ b/iso/bootstrap_admin_node.sh
@@ -347,8 +347,23 @@ fuelmenu --save-only --iface=$ADMIN_INTERFACE || fail
 set +x
 echo "Done!"

+### OPNFV addition BEGIN
+shopt -s nullglob
+for script in /opt/opnfv/bootstrap/pre.d/*.sh
+do
+  echo "Pre script: $script" >> /root/pre.log 2>&1
+  $script >> /root/pre.log 2>&1
+done
+shopt -u nullglob
+
+# Enable sshd
+systemctl enable sshd
+systemctl start sshd
+
+### OPNFV addition END
+
 if [[ "$showmenu" == "yes" || "$showmenu" == "YES" ]]; then
-  fuelmenu || fail
+  fuelmenu
 else
   # Give user 15 seconds to enter fuelmenu or else continue
   echo
@@ -386,8 +401,7 @@ if [ ! -f /etc/fuel_build_id ]; then
 fi

 # Enable sshd
-systemctl enable sshd
-systemctl start sshd
+systemctl restart sshd

 # Enable iptables
 systemctl enable iptables.service
@@ -546,6 +560,16 @@ systemctl start ntpd

 bash /etc/rc.local

+### OPNFV addition BEGIN
+shopt -s nullglob
+for script in /opt/opnfv/bootstrap/post.d/*.sh
+do
+  echo "Post script: $script" >> /root/post.log 2>&1
+  $script >> /root/post.log 2>&1
+done
+shopt -u nullglob
+### OPNFV addition END
+
 if [ "`get_bootstrap_skip`" = "False" ]; then
   build_ubuntu_bootstrap bs_status || true
 else
