--- vnagent_param_setup_ubuntu.sh.orig	2015-12-12 19:58:43.070603884 +0100
+++ vnagent_param_setup_ubuntu.sh.new	2015-12-12 20:08:31.644857358 +0100
@@ -1,5 +1,7 @@
 #! /bin/bash
 
+OS_VER=$1
+
 CFG_FILE=/etc/contrail/agent_param.tmpl
 
 if [ -f $CFG_FILE ] ; then
@@ -15,7 +17,7 @@
 prog=/usr/bin/contrail-vrouter-agent
 echo prog=$prog >> $CFG_FILE
 
-kmod=vrouter
+kmod=/lib/modules/${OS_VER}/updates/dkms/vrouter.ko
 echo kmod=$kmod >> $CFG_FILE
 
 pname=$(basename $prog)
