*** isolinux/isolinux.cfg.orig	2015-04-15 08:29:52.026868322 -0400
--- isolinux/isolinux.cfg	2015-04-15 08:30:34.350868343 -0400
***************
*** 19,22 ****
    menu label Fuel Install (^Static IP)
    menu default
    kernel vmlinuz
!   append initrd=initrd.img biosdevname=0 ks=cdrom:/ks.cfg ip=10.20.0.2 gw=10.20.0.1 dns1=10.20.0.1 netmask=255.255.255.0 hostname=fuel.domain.tld showmenu=no
--- 19,22 ----
    menu label Fuel Install (^Static IP)
    menu default
    kernel vmlinuz
!   append initrd=initrd.img biosdevname=0 ks=cdrom:/ks.cfg ip=10.20.0.2 gw=10.20.0.1 dns1=10.20.0.1 netmask=255.255.255.0 hostname=fuel.domain.tld showmenu=yes

